vault/audit_broker_test.go

```diff
@@ -256,3 +260,58 @@ func TestAuditBroker_Register_MultipleFails(t *testing.T) {
 	require.Error(t, err)
 	require.EqualError(t, err, "vault.(AuditBroker).Register: backend already registered 'b2-no-filter'")
 }
+
+func BenchmarkAuditBroker_File_Request_DevNull(b *testing.B) {
```

Comment:
This test was 'transplanted' from the file backend tests as we don't have a way to manually write to files (we must use the eventlogger/broker).